source/equipment/tests/equipment_api_tests.cpp

```diff
@@ -0,0 +1,204 @@
+/* Copyright © 2001-2019, Canal TP and/or its affiliates. All rights reserved.
+
+This file is part of Navitia,
+    the software to build cool stuff with public transport.
+
+Hope you'll enjoy and contribute to this project,
+    powered by Canal TP (www.canaltp.fr).
+Help us simplify mobility and open public transport:
+    a non ending quest to the responsive locomotion way of traveling!
+
+LICENCE: This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+Stay tuned using
+twitter @navitia
+IRC #navitia on freenode
+https://groups.google.com/d/forum/navitia
+www.navitia.io
+*/
+
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE equipment_test
+
+#include <string>
+#include <vector>
+#include <map>
+
+#include <boost/range/algorithm/sort.hpp>
+#include <boost/test/unit_test.hpp>
+
+#include "ed/build_helper.h"
+#include "equipment/equipment_api.h"
+#include "ptreferential/ptreferential.h"
+#include "utils/logger.h"
+#include "type/type.h"
+#include "type/pt_data.h"
+#include "tests/utils_test.h"
+
+
+using namespace navitia;
+using std::set;
+using std::string;
+using std::map;
+using boost::posix_time::time_period;
+
+struct logger_initialized {
+    logger_initialized() { navitia::init_logger(); }
+};
+BOOST_GLOBAL_FIXTURE( logger_initialized );
+
+namespace {
+
+class Test_fixture {
```

Comment:
Weird case.
Maybe also rename it to EquipmentsTestFixture (not sure about the case here)?

Reply:
:ok_hand: